Sources/PersistentCollections/Node/_Node+isSubset.swift

```diff
@@ -0,0 +1,71 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Collections open source project
+//
+// Copyright (c) 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+extension _Node {
+  /// Returns true if `self` contains a subset of the keys in `other`.
+  /// Otherwise, returns false.
+  @inlinable @inline(never)
+  internal func isSubset<Value2>(
+    _ level: _Level,
+    of other: _Node<Key, Value2>
+  ) -> Bool {
```

Comment:
Consider adding benchmarks for `isSubset` and other structural operations.

Structural operations with persistent data structures have an inherent benefit over performing those operations on a logical level. Hence, it would be great to highlight those differences with benchmarks.